src/domain/collection/curatedEvents/__tests__/CuratedEvents.test.tsx

```diff
@@ -225,19 +228,55 @@ const paginationTest = async ({
 const getMocks = (
   events: EventFieldsFragment[],
   ids = eventIds,
-  type: 'event' | 'course' = 'event'
-) => [
-  {
-    request: {
-      query: EventsByIdsDocument,
-      variables: {
-        ids,
-        include: ['location'],
+  page?: number | undefined,
+  maxPage: number = 1
+) => {
+  let variables = {
+    ids,
+    include: ['location'],
+    pageSize: 10,
+    sort: 'end_time',
+  };
+
+  let meta = {
+    count: ids.length,
+    previous: null,
+    next: null,
+    __typename: 'Meta',
+  };
+
+  if (page && page < maxPage) {
+    meta[
+      'next'
+    ] = `https://api.hel.fi/linkedevents/v1/event/?ids=${ids.toString()}&page_size=10&page=${
+      page + 1
+    }&sort=end_time&include=location`;
+  }
+  if (page > 1) {
+    variables['page'] = page;
```

Comment:
Dot notation could be used here `variables.page = page` (you can set page as undefined in variables object)